docs/INSTALL.md

```diff
@@ -203,7 +201,7 @@ Once you have finished installing the program via one of the two above methods f
 #### Configuring JoinMarket
 Lastly we must configure JoinMarket to allow it to connect to Bitcoin Core. Refer to [this](USAGE.md#managing-your-joinmarket-data) section in the usage guide to generate a `joinmarket.cfg` file using `scripts/wallet-tool.py`.
 
-Edit your `joinmarket.cfg` file (at `~/.joinmarket` in Ubuntu if you used WSL, or in `C:\Users\<your username>\AppData\Roaming\joinmarket` if not) and replace the following lines in the section `[BLOCKCHAIN]`
+Edit your `joinmarket.cfg` file (at `~/.joinmarket` in Ubuntu if you used WSL, or in `C:\Users\<your username>\AppData\Local\joinmarket` if not) and replace the following lines in the section `[BLOCKCHAIN]`
```

Comment:
I don't use windows, but afaik this changed there recently: roaming -> local

Reply:
Depends on a Windows version I guess? On my Windows 10 laptop it's Roaming.

Reply:
yes probably win11,

at least I thought so but a more specific web search says otherwise :) https://www.makeuseof.com/tag/appdata-roaming-vs-local/

my bad, I dropped the change